csun-metalab/nomi
# NAM-513 -- UI: Create line break between first and last name in profile view

# Description
Create a line break between student first and last name.
  
# Testing
1. Visit a student profile and ensure each word is on the next line.
2. Ensure the font-size works with different screen resolutions.
